component_versions.sh

```diff
@@ -21,9 +21,12 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
+# TODO: SAT_CFS_DOCKER_VERSION AND DOCS_SAT_VERSION are unstable,
```

Comment:
I think for this what we'll have to do is wait until the other pull requests merge, and then possibly to a merge to main and then update this file to include proper main branch versions. We could also potentially use the integration branch versions.

Reply:
I agree that we should merge the other PRs first and probably do an integration -> main merge and then update this.